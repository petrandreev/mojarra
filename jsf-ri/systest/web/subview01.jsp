<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
<html>
<head>
<title>subview01</title>
</head>
<body>
<h:outputText value="Begin test <c:import> with subview tag in imported page"/>

<p>
<c:import url="foo01.jsp"/>
</p>

<p><h:outputText value="subview01"/></p>

<p><c:import url="bar01.jsp"/></p>

<p><h:outputText value="End test <c:import> with subview tag in imported page"/></p>
</body>
</html>
</f:view>
