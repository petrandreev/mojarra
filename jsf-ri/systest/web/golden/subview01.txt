<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->






<html>
<head>
<title>subview01</title>
</head>
<body>
Begin test &lt;c:import&gt; with subview tag in imported page
foo01<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->









subview01
bar01<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->









End test &lt;c:import&gt; with subview tag in imported page
</body>
</html>

